lib/FrontendTool/FrontendTool.cpp

```diff
@@ -291,8 +291,9 @@ static bool emitSyntax(SourceFile *SF, LangOptions &LangOpts,
 
 /// Writes SIL out to the given file.
 static bool writeSIL(SILModule &SM, ModuleDecl *M, bool EmitVerboseSIL,
-                     StringRef OutputFilename, bool SortSIL) {
-  auto OS = getFileOutputStream(OutputFilename, M->getASTContext());
+                     bool SortSIL) {
+  auto OS =
+      getFileOutputStream(SM.getPSPs().OutputFilename, M->getASTContext());
```

Comment:
Yeah, in theory you ought to be able to write the same SIL module multiple places. I can't see why someone would ever do that, but it shouldn't be baked in.